### PR TITLE
feat(transaction): adding warning banner about missing transaction usage

### PR DESCRIPTION
**Pull request by jusigler**
adding a banner to let people know that the graph doesn't include transaction and attachment usage yet.

### CODE REVIEW

**Comment by jusigler** [comment]
And I just realized that I need to put feature flag there so that the banner only shows up for am1 subscription plan

File: src/sentry/static/sentry/app/views/organizationStats/organizationStatsDetails.jsx [hunk 1]
```diff
@@ -95,6 +96,9 @@ class OrganizationStats extends React.Component {
           )}
         </div>
         <div>
+          <Alert type="info" icon="icon-circle-info">
+            {t('Transactions and attachments are not present in the graph currently.')}
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/views/organizationStats/organizationStatsDetails.jsx:100]: Is this the phrasing we want to use?

Should we make transactions a link?

> Transactions and attachments are not yet included in the graph

Feels a little more on par with our tone.

**Review by evanpurkhiser — APPROVED** [review]

File: src/sentry/static/sentry/app/views/organizationStats/organizationStatsDetails.jsx (hunk 1 above)
Comment (jmanhart) [reply src/sentry/static/sentry/app/views/organizationStats/organizationStatsDetails.jsx:100]: 👍

**Review by maheskett — DISMISSED** [review]
could we only display this warning if the organization has access to transactions and/or attachments

**Review by markstory — APPROVED** [review]

**Pull request merged**